blockly-core/core/ui/contract_editor/contract_editor.js

```diff
@@ -67,15 +92,15 @@ Blockly.ContractEditor.EXAMPLE_BLOCK_TYPE = 'functional_example';
 Blockly.ContractEditor.EXAMPLE_BLOCK_ACTUAL_INPUT_NAME = 'ACTUAL';
 
 Blockly.ContractEditor.typesToColorsHSV = {
-  'none': [0, 0, 0.6],
+  // 'none': [0, 0, 0.6], // not available to users
   'Number': [192, 1.00, 0.99], // 00ccff
   'string': [180, 1.00, 0.60], // 0099999
   'image': [285, 1.00, 0.80], // 9900cc
   'boolean': [90, 1.00, 0.4] // 336600
 };
```

Comment:
Since it looks like everything in https://github.com/code-dot-org/code-dot-org/blob/fix_variables_view/apps/src/functionalBlockUtils.js#L42-42 is blockly-core specific with no blockly dependencies, I'll move that all into a blockly-core class